shinydashboard/lantern/ui.R

```diff
@@ -0,0 +1,67 @@
+# Define base user interface
+
+ui <- dashboardPage(
+  dashboardHeader(
+    title = "Lantern Dashboard",
+    titleWidth = 200
+  ),
+  # Sidebar with a slider input for number of bins 
```

Comment:
It looks like the comments in this file are out of date?

Reply:
Updated comment